pkg/META

```diff
@@ -1,8 +1,8 @@
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
here you can use the watermarking feature `%%VERSION_NUM%%` (which topkg distrib then fills in) :)